.github/workflows/urlchecker.yml

```diff
@@ -21,7 +21,7 @@ jobs:
         retry_count: 3
 
         # White listed patterns (seem to have SSL issues but work in browser)
-        white_listed_patterns: sc19.supercomputing.org,https://pace.gatech.edu
+        white_listed_patterns: sc19.supercomputing.org,https://pace.gatech.edu,github/workspace/_posts/newsletters/
 
```

Comment:
@danielskatz you added to the wrong line (not the one that I linked) this is `white_listed_patterns` (for urls) and I linked you to the last one, `white_listed_files`.

Reply:
ok, thanks - will fix